misc/scripts/install-local.sh

```diff
@@ -216,6 +255,8 @@ if [[ $NOBUILDDEP -eq 0 ]]; then
 	fi
 fi
 
+makeVirtualDeb
```

Comment:
The function should only be called if the package is not a `-deb`
Use `if [[ "$url" == *".deb" ]]`